### PR TITLE
DOC user guide on the hub

### PR DESCRIPTION
**Pull request by adrinjalali**
A little bit of user guide and improvements to docs.

### CODE REVIEW

**Comment by adrinjalali** [comment]
Ready for review @skops-dev/maintainers

File: docs/index.rst
Comment (BenjaminBossan) [inline docs/index.rst:9]: How about linking `skops` to github here? AFAICT, there is no link to the repo anywhere in the documentation at all.

**Review by BenjaminBossan — APPROVED** [review]
Great addition, thanks. I proposed a change, but it's not critical at all.

File: docs/index.rst
Comment (adrinjalali) [reply docs/index.rst:9]: Done!

**Comment by adrinjalali** [comment]
Also wondering what @osanseviero thinks about this.

**Comment by BenjaminBossan** [comment]
> Also wondering what osanseviero thinks about this.

Okay, I'll wait before merging then.

File: docs/hf_hub.rst
Comment (osanseviero) [inline docs/hf_hub.rst:7]: I think you could also mention other benefits apart from the API
* A widget to try the model directly in the browser
* Metadata tags for discoverability
* Collaborating with others in a model
* Downloading/persisting in a way easy to share

File: docs/index.rst
Comment (osanseviero) [inline docs/index.rst:9]:
```suggestion
``skops`` is a Python library helping you share your `scikit-learn
```

**Review by osanseviero — APPROVED** [review]
Looks good! I just left minor suggestions. It would be great if @merveenoyan could take a quick look on Monday as well unless we're in a hurry to merge this

**Comment by osanseviero** [comment]
Very friendly ping @merveenoyan as I realized last ping was during holidays+before weekend, so I wanted to resurface this.

File: docs/hf_hub.rst
Comment (merveenoyan) [inline docs/hf_hub.rst:79]: Maybe an emphasis on this with bold font would be noice but LGTM 😅 it's one of the main value propositions of skops that's why
image

**Review by merveenoyan — APPROVED** [review]
Left minor comments, feel free to ignore if you don't agree but LGTM

File: docs/hf_hub.rst
Comment (adrinjalali) [reply docs/hf_hub.rst:13]: through what? I'm not sure how to finish the sentence

Comment (merveenoyan) [reply docs/hf_hub.rst:13]: Having repositories on Hub in general and pull requests + discussions.

**Pull request merged**
